content/en/docs/tasks/federation/_index.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: "Federation"
+title: "Federation v1"
```

Comment:
I think with the “v1” omitted this would be OK to merge; you can then open a separate PR about adding the “v1” here and we can discuss that bit separately.

How does that sound?

Reply:
I agree with @tengqm said, modification with federation v1 is meaningless, it's been deprecated, and from my real world experience, the kubefed, which is indicated to federation v2, is also in development stage and NOT be verified in real world ( just from my personal sight ) , that is the reason why our kubefed project owner is NOT active to build document in our website.

IMO,  I'd like to move on the other PR, #17949 , just leave some deprecation message about federation v1, kill the others left, and leave some guides to federation v2.